products/revenue_analytics/backend/hogql_queries/test/test_revenue_analytics_growth_rate_query_runner.py

```diff
@@ -0,0 +1,97 @@
+from freezegun import freeze_time
+from pathlib import Path
+from datetime import date
+from decimal import Decimal
+
+from products.revenue_analytics.backend.hogql_queries.revenue_analytics_growth_rate_query_runner import (
+    RevenueAnalyticsGrowthRateQueryRunner,
+)
+from products.revenue_analytics.backend.models import STRIPE_DATA_WAREHOUSE_CHARGE_IDENTIFIER
+from posthog.schema import (
+    DateRange,
+    RevenueAnalyticsGrowthRateQuery,
+    RevenueAnalyticsGrowthRateQueryResponse,
+)
+from posthog.test.base import (
+    APIBaseTest,
+    ClickhouseTestMixin,
+    snapshot_clickhouse_queries,
+)
+from posthog.warehouse.models import ExternalDataSchema
+
+from posthog.warehouse.test.utils import create_data_warehouse_table_from_csv
+from products.revenue_analytics.backend.hogql_queries.test.data.structure import (
+    REVENUE_TRACKING_CONFIG,
+    STRIPE_CHARGE_COLUMNS,
+)
+
+TEST_BUCKET = "test_storage_bucket-posthog.revenue_analytics.growth_rate_query_runner.stripe_charges"
+
+
+@snapshot_clickhouse_queries
+class TestRevenueAnalyticsGrowthRateQueryRunner(ClickhouseTestMixin, APIBaseTest):
+    QUERY_TIMESTAMP = "2025-02-15"
```

Comment:
Timestamp doesn't do anything now, will come back and actually make it work.

The catch here is that a lot of the charts we have doesn't make a lot of sense with arbitrary dates, so I'll need to rework this anyway in my follow-up PRs